src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj

```diff
@@ -1,9 +1,6 @@
 ﻿
 <Project Sdk="Microsoft.NET.Sdk">
-    <!-- Please do not move/edit code below this line -->
-    <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-    <!-- Please do not move/edit code below this line -->
-
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
Looks like these are breaking changes. Though `0.1.1-preview` version has been published, please consider bumping the minor version (`0.2.0-preview`)